calypso/service.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // Used for tests
-var calypsoID onet.ServiceID
+var CalypsoID onet.ServiceID
```

Comment:
should be kept private if it's only used for test, the test is in the same package, so that should be fine, right?